packages/manager/modules/user-contracts/src/index.js

```diff
@@ -0,0 +1,34 @@
+import angular from 'angular';
+
+import { includes, isEmpty, isArray } from 'lodash';
+import '@ovh-ux/manager-core';
+import '@ovh-ux/ui-kit';
+import '@uirouter/angularjs';
+import 'angular-translate';
```

Comment:
As you use `@ngTranslationInject` annotation I think we should import `@ovh-ux/ng-translate-async-loader` and add `'ngTranslateAsyncLoader'` in AngularJS module dependencies list.

```suggestion
import 'angular-translate';
import '@ovh-ux/ng-translate-async-loader';
```